src/main/java/uk/gov/hmcts/reform/ccd/documentam/client/RestTemplateHeaderModifierInterceptor.java

```diff
@@ -36,6 +38,8 @@ private HttpHeaders getRequestHeaders() {
         HttpHeaders headers = securityUtils.serviceAuthorizationHeaders();
         headers.set(Constants.USERID, securityUtils.getUserInfo().getUid());
         headers.setContentType(MediaType.APPLICATION_JSON);
+
+        log.debug("downstream request headers: {}", headers);
```

Comment:
printing auth headers? This can be done only in lower envs if really needed.

Reply:
Ah, ok. I appreciate log levels will restrict this as you set them by default to INFO.
However, this line can be dangerous to keep. We should revert this PR once diagnostic data from a lower environment is collected.

Reply:
removed this line. If needed for further analyses, I will add it back.